<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sceneEditorTab</class>
 <widget class="QDialog" name="sceneEditorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../MUSEN/QT_GUI/qt_gui.qrc">
    <normaloff>:/QT_GUI/Resources/MUSEN_Ico.png</normaloff>:/QT_GUI/Resources/MUSEN_Ico.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0">
   <item>
    <widget class="QGroupBox" name="groupBoxRotation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Scene rotation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="labelAngle">
        <property name="text">
         <string>Rotation angle</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEditAngleY">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="rotateSystem">
        <property name="toolTip">
         <string>Rotate system around specified point</string>
        </property>
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditAngleX">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelAngleY">
        <property name="text">
         <string>Alpha Y [°]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEditAngleZ">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelAngleZ">
        <property name="text">
         <string>Alpha Z [°]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditCenterY">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditCenterZ">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelRotation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Center of rotation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="undoRotation">
        <property name="toolTip">
         <string>Undo last rotation</string>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelCenterY">
        <property name="text">
         <string>Y [m]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelCenterX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>X [m]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelCenterZ">
        <property name="text">
         <string>Z [m]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditCenterX">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="setCenterOfMass">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set center of mass as rotation center</string>
        </property>
        <property name="text">
         <string>Center of mass</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelAngleX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Alpha X [°]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMove">
     <property name="title">
      <string>Scene movement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="labelOffsetX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>X [m]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelOffset">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelOffsetY">
        <property name="text">
         <string>Y [m]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelOffsetZ">
        <property name="text">
         <string>Z [m]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditOffsetY">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditOffsetX">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditOffsetZ">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="moveSystem">
        <property name="toolTip">
         <string>Move system</string>
        </property>
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxVelocity">
     <property name="title">
      <string>Scene velocity</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="labelVelocity">
        <property name="text">
         <string>New velocity </string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelVelocityZ">
        <property name="text">
         <string>Vz [m/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditVelocityZ">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="setVelocity">
        <property name="toolTip">
         <string>Set initial velocity of all objects in the system</string>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditVelocityX">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelVelocityX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Vx [m/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEditVelocityY">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelVelocityY">
        <property name="text">
         <string>Vy [m/s]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBoxPBC">
       <property name="title">
        <string>Periodic boundary conditions</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>3</number>
        </property>
        <item row="1" column="4">
         <widget class="QLineEdit" name="lineEditVelX"/>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="lineEditPBCMaxZ"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBoxPBCX">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEditPBCMinY"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBoxPBCY">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEditPBCMaxY"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelPBCMin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Min [m]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelPBCMax">
          <property name="text">
           <string>Max [m]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEditPBCMaxX"/>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBoxPBCZ">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditPBCMinX"/>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="lineEditVelZ"/>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="lineEditVelY"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEditPBCMinZ"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="labelPBCVel">
          <property name="text">
           <string>Velocity [m/s]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOther">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxAnisotropy">
        <property name="toolTip">
         <string>Consider anisotropy of non-spherical objects</string>
        </property>
        <property name="text">
         <string>Consider particles anisotropy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxContactRadius">
        <property name="text">
         <string>Contact radius of particles differs from their radius</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonResetBonds">
          <property name="toolTip">
           <string>Set initial bond length of all bonds equal to their actual length at time point 0</string>
          </property>
          <property name="statusTip">
           <string>Set initial bond length of all bonds equal to their actual length at time point 0</string>
          </property>
          <property name="whatsThis">
           <string>Set initial bond length of all bonds equal to their actual length at time point 0</string>
          </property>
          <property name="text">
           <string>Reset initial bond length</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>370</width>
            <height>11</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>setCenterOfMass</tabstop>
  <tabstop>lineEditCenterX</tabstop>
  <tabstop>lineEditCenterY</tabstop>
  <tabstop>lineEditCenterZ</tabstop>
  <tabstop>undoRotation</tabstop>
  <tabstop>lineEditAngleX</tabstop>
  <tabstop>lineEditAngleY</tabstop>
  <tabstop>lineEditAngleZ</tabstop>
  <tabstop>rotateSystem</tabstop>
  <tabstop>lineEditOffsetX</tabstop>
  <tabstop>lineEditOffsetY</tabstop>
  <tabstop>lineEditOffsetZ</tabstop>
  <tabstop>moveSystem</tabstop>
  <tabstop>lineEditVelocityX</tabstop>
  <tabstop>lineEditVelocityY</tabstop>
  <tabstop>lineEditVelocityZ</tabstop>
  <tabstop>setVelocity</tabstop>
  <tabstop>pushButtonOk</tabstop>
 </tabstops>
 <resources>
  <include location="../../MUSEN/QT_GUI/qt_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>sceneEditorTab</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
